tests/components/awair/test_sensor.py

```diff
@@ -176,7 +177,7 @@ def alter_time(retval):
     sensor = hass.states.get("sensor.awair_humidity")
     assert sensor.state == "32.7"
     assert sensor.attributes["device_class"] == DEVICE_CLASS_HUMIDITY
-    assert sensor.attributes["unit_of_measurement"] == "%"
+    assert sensor.attributes["unit_of_measurement"] == UNIT_PERCENTAGE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093397)